airbyte-cdk/python/unit_tests/sources/declarative/test_concurrent_declarative_source.py

```diff
@@ -60,6 +66,41 @@
         )
     ]
 )
+_LOCATIONS_RESPONSE = HttpResponse(json.dumps([
+    {"id": "444", "name": "Yongen-jaya", "updated_at": "2024-08-10"},
+    {"id": "scramble", "name": "Shibuya", "updated_at": "2024-08-10"},
+    {"id": "aoyama", "name": "Aoyama-itchome", "updated_at": "2024-08-10"},
+    {"id": "shin123", "name": "Shinjuku", "updated_at": "2024-08-10"},
+]))
+_PARTY_MEMBERS_RESPONSE = HttpResponse(json.dumps([
+    {"id": "amamiya", "first_name": "ren", "last_name": "amamiya", "updated_at": "2024-07-10"},
+    {"id": "nijima", "first_name": "makoto", "last_name": "nijima", "updated_at": "2024-08-10"},
+    {"id": "yoshizawa", "first_name": "sumire", "last_name": "yoshizawa", "updated_at": "2024-09-10"},
+]))
+_PALACES_RESPONSE = HttpResponse(json.dumps([
+    {"id": "0", "world": "castle", "owner": "kamoshida"},
+    {"id": "1", "world": "museum", "owner": "madarame"},
+    {"id": "2", "world": "bank", "owner": "kaneshiro"},
+    {"id": "3", "world": "pyramid", "owner": "futaba"},
+    {"id": "4", "world": "spaceport", "owner": "okumura"},
+    {"id": "5", "world": "casino", "owner": "nijima"},
+    {"id": "6", "world": "cruiser", "owner": "shido"},
+]))
+_PARTY_MEMBERS_SKILLS_RESPONSE = HttpResponse(json.dumps([
+    {"id": "0", "name": "hassou tobi"},
+    {"id": "1", "name": "mafreidyne"},
+    {"id": "2", "name": "myriad truths"},
+]))
+_EMPTY_RESPONSE = HttpResponse(json.dumps([]))
+_NOW = "2024-09-10T00:00:00"
+_DECLARATIVE_CURSOR_PARTY_MEMBERS_SLICES = [
+    {"start": "2024-06-26", "end": "2024-07-10"},
+    {"start": "2024-07-11", "end": "2024-07-25"},
+    {"start": "2024-07-26", "end": "2024-08-09"},
+    {"start": "2024-08-10", "end": "2024-08-24"},
+    {"start": "2024-08-25", "end": "2024-09-08"},
+    {"start": "2024-09-09", "end": "2024-09-10"},
+]  # FIXME can be removed/updated once the declarative cursor is updated to match the concurrent cursor behavior on lookback windows
```

Comment:
@brianjlai for the change in the `DatetimeBasedCursor`, it will probably affect this variable

Reply:
yep expected. i fixed the lookback functionality yesterday (which technically didn't affect concurrent since its doing the right thing), but if i see test failures i'll fix